src/main/java/uk/gov/hmcts/reform/translate/service/DictionaryService.java

```diff
@@ -58,6 +61,27 @@ public Map<String, String> getDictionaryContents() {
         }
     }
 
+    public Map<String, String> getTranslations(@NonNull final Set<String> phrases) {
```

Comment:
As this performs a read followed by a possible save should it have a transaction?

Reply:
Doing this in WLTS-24

Reply:
Resolved in #42 / #40